src/cmds/scripts/pbs.service.in

```diff
@@ -55,7 +55,7 @@ DefaultDependencies=true
 [Service]
 Type=forking
 Restart=no
-TimeoutSec=5min
+TimeoutSec=0
```

Comment:
I assume this means that the timeout is disabled. Perhaps a larger value would make more sense? I'm not certain that waiting indefinitely is the correct behavior.